src/app/modules/main/communities/controller.nim

```diff
@@ -44,6 +44,10 @@ method init*(self: Controller) =
 method getAllCommunities*(self: Controller): seq[CommunityDto] =
   result = self.communityService.getAllCommunities()
 
+  self.events.on(SIGNAL_COMMUNITY_CATEGORY_REORDERED) do(e:Args):
```

Comment:
remove this as it is unused? also maybe emitting signal is not needed? handled in chat_sections controller